### PR TITLE
[INJIWEB-511]: Add Gradient to Icon 

### PR DESCRIPTION
**Pull request by kamlesh012**

### CODE REVIEW

File: inji-web/src/components/Common/LanguageSelector.tsx [hunk 1]
```diff
@@ -45,12 +44,12 @@ export const LanguageSelector: React.FC = () => {
                 onMouseDown={() => setIsOpen(open => !isOpen)}>
                 <p data-testid={`Language-Selector-Selected-DropDown-${language}`}>{LanguagesSupported.find(lang => lang.value === language)?.label}</p>
                 {isOpen ?
-                    <GradientWrapper>
```
Comment (vijay151096) [inline inji-web/src/components/Common/LanguageSelector.tsx:48]: why is GradientWrapper is removed  ?a

File: inji-web/src/components/Common/LanguageSelector.tsx [hunk 2]
```diff
@@ -45,12 +44,12 @@ export const LanguageSelector: React.FC = () => {
...
+                   
                         <RiArrowUpSFill size={20} color={'var(--iw-color-languageArrowIcon)'} />
-                    </GradientWrapper> :
-                    <GradientWrapper>
+                    :
+                    
                         <RiArrowDownSFill size={20} color={'var(--iw-color-languageArrowIcon)'}/>
-                    </GradientWrapper>}
```
Comment (vijay151096) [inline inji-web/src/components/Common/LanguageSelector.tsx:53]: why is GradientWrapper is removed  ?a

Comment (kamlesh012) [reply inji-web/src/components/Common/LanguageSelector.tsx:53]: added it back again, forget to readd it on the arrow.

File: inji-web/src/components/Common/LanguageSelector.tsx (hunk 1 above)
Comment (kamlesh012) [reply inji-web/src/components/Common/LanguageSelector.tsx:48]: re added it, removed by mistake

**Pull request merged**